index.html

```diff
@@ -125,14 +168,20 @@ <h1>Tonic</h1>
                </ul>
                <div class="external_link">
                    <a href="#">See Project</a>
-               </div>
+               
```

Comment:
- [ ]  Kyndly stick to the figma design as much as posible, in the cards section you have to place the card images according to the figma design, please notice that in some cards the image is on the right side and in others the image is on the left side.

## Figma design
![image](https://user-images.githubusercontent.com/95243996/201198370-71ea6965-b9d1-4b5d-beb6-f78b8a7fbf75.png)

## Your design 

![image](https://user-images.githubusercontent.com/95243996/201198844-9dfd5f22-7c3b-4fed-9c83-09500fdb1351.png)